Bottom: cd2bee162418bbb57796de01c9211dc4a4977a09
Top:    3812d291a01da883ee0fa1ff7629bc303f359cde
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-15 11:05:23 -0700

Refresh of fix-sip

---
diff --git a/lldb/source/Core/Debugger.cpp b/lldb/source/Core/Debugger.cpp
index dbc8e3be834..18397d00dca 100644
--- a/lldb/source/Core/Debugger.cpp
+++ b/lldb/source/Core/Debugger.cpp
@@ -974,7 +974,6 @@ void Debugger::AdoptTopIOHandlerFilesIfInvalid(FileSP &in, StreamFileSP &out,
   IOHandlerSP top_reader_sp(m_input_reader_stack.Top());
   // If no STDIN has been set, then set it appropriately
   if (!in || !in->IsValid()) {
-    in.reset();
     if (top_reader_sp)
       in = top_reader_sp->GetInputFileSP();
     else
@@ -985,7 +984,6 @@ void Debugger::AdoptTopIOHandlerFilesIfInvalid(FileSP &in, StreamFileSP &out,
   }
   // If no STDOUT has been set, then set it appropriately
   if (!out || !out->GetFile().IsValid()) {
-    out.reset();
     if (top_reader_sp)
       out = top_reader_sp->GetOutputStreamFileSP();
     else
@@ -996,7 +994,6 @@ void Debugger::AdoptTopIOHandlerFilesIfInvalid(FileSP &in, StreamFileSP &out,
   }
   // If no STDERR has been set, then set it appropriately
   if (!err || !err->GetFile().IsValid()) {
-    err.reset();
     if (top_reader_sp)
       err = top_reader_sp->GetErrorStreamFileSP();
     else
